Rakefile

```diff
@@ -4,9 +4,6 @@ require 'fileutils'
 require 'colorize'
 
 task :release_check do
-  puts "Linting code".colorize(:cyan)
-  system("swiftlint") || abort
```

Comment:
Do we want to keep this?

Reply:
I think it's fine to get rid of it since we are running the linter on the PRs before they get merged.